Dataset.py

```diff
@@ -555,11 +582,15 @@ def finish_epoch(self):
     This would get called at the end of the epoch (currently optional only).
     After this, further calls to :func:`get_data` or :func:`load_seqs` are invalid,
     until a new call to :func:`init_seq_order` follows.
+
+    TODO: currently this seems to be important only for the SprintDataset
     """
     self.epoch = None
 
   def get_current_seq_order(self):
     """
+    TODO: This function is only used by the MetaDataset, so this might be flagged as protected
```

Comment:
No, this is not just used via `self`, but also for other datasets (externally). Protected functions/attributes are mostly just used via `self`. This is an example which I would make public.